packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -731,8 +731,6 @@ class ThemeData extends Diagnosticable {
   /// Linearly interpolate between two themes.
   ///
   /// {@template flutter.material.themeData.lerp}
```

Comment:
we should probably move this to the first inclusion of this text in the animation layer... it's weird that the animation layer refers to documentation in the material layer.

Reply:
I couldn't think of an obvious place, but you're right, it is weird to put it in `material/`.

Moved `painting/gradient.dart`

See: https://github.com/flutter/flutter/pull/22388/